.github/workflows/check-release.yaml

```diff
@@ -0,0 +1,18 @@
+on: push
+
+name: R-CMD-check
+
+jobs:
+  R-CMD-check:
+    runs-on: macOS-latest
```

Comment:
Is `mac` really the preferred target OS for this? I would have assumed it would be `ubuntu`.